README.md

```diff
@@ -36,55 +36,90 @@ class TinySpacenetExperimentSet(rv.ExperimentSet):
         train_label_uri = '{}/buildings_AOI_2_Vegas_img205.geojson'.format(base_uri)
         val_image_uri = '{}/RGB-PanSharpen_AOI_2_Vegas_img25.tif'.format(base_uri)
         val_label_uri = '{}/buildings_AOI_2_Vegas_img25.geojson'.format(base_uri)
+        channel_order = [0, 1, 2]
 
-        task = rv.TaskConfig.builder(rv.OBJECT_DETECTION) \
+        # ------------- TASK -------------
+
+        task = rv.TaskConfig.builder(rv.SEMANTIC_SEGMENTATION) \
                             .with_chip_size(512) \
                             .with_classes({
                                 'building': (1, 'red')
                             }) \
-                            .with_chip_options(neg_ratio=1.0,
-                                               ioa_thresh=0.8) \
-                            .with_predict_options(merge_thresh=0.1,
-                                                  score_thresh=0.5) \
+                            .with_chip_options(
+                                chips_per_scene=9,
+                                debug_chip_probability=1.0,
+                                negative_survival_probability=0.25,
+                                target_classes=[1],
+                                target_count_threshold=1000) \
```

Comment:
I would change these to try to use more default options to shorten the example and not confuse people.

Reply:
Done.